x-pack/plugins/infra/public/components/asset_details/content/content.tsx

```diff
@@ -30,25 +29,26 @@ export const Content = ({
     onTabsStateChange(state);
   };
 
+  const timeRangeTs = toTimestampRange(timeRange);
   return (
     <>
       <TabPanel activeWhen={FlyoutTabIds.ANOMALIES}>
         <Anomalies nodeName={node.name} onClose={overrides?.anomalies?.onClose} />
       </TabPanel>
       <TabPanel activeWhen={FlyoutTabIds.OVERVIEW}>
         <Overview
-          currentTimeRange={currentTimeRange}
+          dateRange={timeRange}
```

Comment:
Q: Is this value the same as the one in `overrides?.overview?.dateRange` 🤔

Reply:
I've removed the `overrides?.overview?.dateRange` and centralized it in the `dateRange` that the asset details receives: https://github.com/elastic/kibana/pull/161559/files#diff-38508a9b3414e92ef6d99e680bd5a730697857523d9aa782da568f6d56f6f8d8R35

In the end I think we need only one `dateRange` to be passed to the component.

Reply:
the name of the variable here should be `dateRange` actually